PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
-<!-- Please explain testing planned this can include
-automated testing, manual testing, cross browser testing -->
 
 ## Screenshots
-<!-- Please provide screenshots of UI changes if applicable -->
-
-## Acceptance Criteria (Definition of Done)
 
-#### Unique to this PR
-<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
```

Comment:
This comment seems helpful, to provide context for the larger body of work related to the current PR. If this section is removed, should the additional guidance be relocated to acceptance criteria?

Reply:
This has been moved to AC, yes.

Acceptance criteria = unique to this PR
Definition of done = applicable to all PRs

Reply:
Thanks for clarifying.

Reply:
I mused about this about in [this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/8301#discussion_r215045989).